todo/src/Todo.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import './TodoList.css'
+
+import { connect } from 'react-redux';
+import { toggle } from './actions';
+import { removeTodo } from './actions';
+
+const Todo = (props) => {
+
+  return(
+    <div>
+      <li style={props.completed ? {textDecoration: 'line-through'} : null} onClick={() => {props.toggle(props.id)}}>{props.todo}</li>
+      <button onClick={() => {props.removeTodo(props.id)}}>x</button>
+    </div>
+  );
+}
+
+const mapStateToProps = state =>{
+  return{
+    todos: state.todos
```

Comment:
Do you need to map all of your todos to the props for every Todo component? Do you even need to map state to props in this component?